src/System.Net.WebSockets.Client/src/MatchingRefApiCompatBaseline.uap.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Net.WebSockets.Client:
```

Comment:
What is this file for?  It is related to UAP?  Why are these errors generated?

Reply:
I believe the warning happens because the ref for uap doesn't contain the new method but the implementation does. @weshaggard, is that right?

Reply:
>because the ref for uap doesn't contain the new method

Why wouldn't UAP contain the new method?  UAP should basically be the same as netcoreapp reference assembly.  There were not any diversions between reference assemblies of UAP vs netcoreapp before.

Reply:
>I believe the warning happens because the ref for uap doesn't contain the new method but the implementation does. @weshaggard, is that right?

That is right. 

I would expect these to be in uap as well. I don't see anything in the ref build that conditions it for uap or netcoreapp. @stephentoub perhaps you hit this before the ref was updated and built for uap? I would suggest removing it and seeing if CI passes.

Reply:
Thanks, Wes.  Whatever the issue was appears to have been fixed in the interim.  I've removed the suppression file.